components/Footer/index.js

```diff
@@ -0,0 +1,17 @@
+import * as Styled from './styles';
+
+function index() {
```

Comment:
Interesante que deje poner el nombre en minúscula, qué te parece cambiarlo a 
```suggestion
function Footer() {
```
No habrá colisión con la línea 5 porque es `<Styled.Footer>` :D